tools/graphmap/graphmap_align.xml

```diff
@@ -0,0 +1,65 @@
+<tool id="graphmap_align" name="Mapper" version="0.5.2">
+    <description>for long, error-prone reads, like Nanopore ONT and PacBio.</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="exit_code">
+    <![CDATA[
+        ln -s '$r' ref.${r.ext} && 
+        ln -s '$d' reads.${d.ext} && 
+
+        graphmap align  
+        -r ref.${r.ext} 
+        -d reads.${d.ext}
+        -o output.sam 
+        #if $preset:
+            -x $preset
+        #end if
+        -t \${GALAXY_SLOTS:-4} 
+    ]]></command>
+    <inputs>
+        <param argument="-r" type="data"  format="fasta,fastq" label="reference sequence" />
+        <param argument="-d" type="data"  format="fasta,fastq" label="reads" />
+        <param argument="-u" type="boolean" truevalue="-u" falsevalue="" label="SAM in the order of input reads" />
+
+        <param name="preset" type="select" optional="True" label="Pre-set parameters (-x)">
```

Comment:
A short description what they are doing ... maybe? Not required for a first version I guess.

Reply:
added